service/src/main/kotlin/io/provenance/digitalcurrency/consortium/stream/EventStreamConsumer.kt

```diff
@@ -216,11 +217,14 @@ class EventStreamConsumer(
                 // val toAddressBankUuid =
                 //     pbcService.getAttributeByTagName(event.toAddress, bankClientProperties.kycTagName)?.bankUuid()
 
-                // persist a record of this transaction if either the from or the to address has this bank's attribute
-                if (toAddressBankUuid != null || fromAddressBankUuid != null) {
-                    TransferWrapper(event, toAddressBankUuid, fromAddressBankUuid)
-                } else {
-                    null
+                when {
+                    // persist a record of this transaction if either the from or the to address has this bank's attribute
+                    toAddressBankUuid != null || fromAddressBankUuid != null ->
+                        TransferWrapper(event, toAddressBankUuid, fromAddressBankUuid)
+                    // persist a record of this transaction if the from or the to is this bank's address
+                    event.fromAddress == pbcService.managerAddress || event.toAddress == pbcService.managerAddress ->
```

Comment:
not 100% sold on needing this for both directions since toAddress matching bank address is somewhat duplicative of the redemptions

Reply:
@scirner22 mind looking at this piece specifically - want to make sure this isnt against your decisions here